.changeset/grumpy-snakes-smell.md

```
@@ -0,0 +1,5 @@
+---
+'@mastra/elasticsearch': patch
+---
+
+Corrected the minimum supported @mastra/core version to match the APIs used by this store.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)